unstable/wlr-screencopy-unstable-v1.xml

```diff
@@ -79,7 +80,7 @@
     </request>
   </interface>
 
-  <interface name="zwlr_screencopy_frame_v1" version="1">
+  <interface name="zwlr_screencopy_frame_v1" version="2">
```

Comment:
The version is inherited from the object that created the child object. Thus all interface versions in the protocol needs to be bumped.

Reply:
Hmm, actually, I changed it in the implementation so that the version comes from the interface struct. Doesn't that make more sense?

Reply:
No, because then the object will always have the same version, regardless of what version is specified by the client.